sdl_android/src/main/java/com/smartdevicelink/util/AndroidTools.java

```diff
@@ -106,9 +106,10 @@ public static boolean isServiceExported(Context context, ComponentName name) {
 					try {
 						packageInfo = packageManager.getPackageInfo(info.serviceInfo.packageName, 0);
 					} catch (NameNotFoundException e) {
-					} finally {
-						sdlAppInfoList.add(new SdlAppInfo(info, packageInfo));
+						//Package was not found, likely a sign the resolve info can't be trusted.
 					}
+					sdlAppInfoList.add(new SdlAppInfo(info, packageInfo));
```

Comment:
do we want to wrap this in a null check, or add null package info's in?

Reply:
From my understanding the `getPackageInfo` method doesn't return null. It will instead throw an exception if the package can't be found. [Reference](https://developer.android.com/reference/android/content/pm/PackageManager.html#getPackageInfo)